src/domain/permissions.spec.js

```diff
@@ -10,6 +10,7 @@ describe('permissions', () => {
 
     const permissions = Object.entries(sut)
       .filter(entry => (/^[A-Z_]+$/).test(entry[0]))
+      .filter(entry => entry[1] !== LIST_EXPORTABLE_CONTENT)
```

Comment:
Could we maybe also have a test for the case when this permission **is** set?

Reply:
Ha :laughing: right

Reply:
Done!